### PR TITLE
[TASK] Setup postgresql as db instead

### PR DESCRIPTION
**Pull request by Diezagu**
I have changed the database system to PostgreSQL :)

### CODE REVIEW

**Review by bryanmg — APPROVED** [review]

File: config/database.yml [hunk 1]
```diff
@@ -5,21 +5,21 @@
...
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
 
 development:
   <<: *default
-  database: db/development.sqlite3
+  database: db/development
```
Comment (itsmeurbi) [inline config/database.yml]: Let's use a more descriptive db name. Maybe something like medial_control_env:
(i.e. medial_control_development, medial_control_test and medial_control_production)

**Review by itsmeurbi — CHANGES_REQUESTED** [review]

**Review by itsmeurbi — APPROVED** [review]

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
You didn't complete the installation process. Missing changes on Gemfile.lock
`bundle install` pending

**Review by itsmeurbi — CHANGES_REQUESTED** [review]
Ok, now PR seems to be legit. But it should be one single commit.
Take a look at how [rebase interactive](https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history) works

**Review by itsmeurbi — APPROVED** [review]
Seems to be legit!

**Pull request merged**